12a_fix_thumbpdf_invocation.dpatch  <preining@logic.at>
Fix invocation of thumbpdf to get rid of magic
---
 texk/texlive/linked_scripts/thumbpdf/thumbpdf.pl |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

Index: texlive-bin-2009~svn13478/texk/texlive/linked_scripts/thumbpdf/thumbpdf.pl
===================================================================
--- texlive-bin-2009~svn13478.orig/texk/texlive/linked_scripts/thumbpdf/thumbpdf.pl	2009-05-26 11:00:08.000000000 +0200
+++ texlive-bin-2009~svn13478/texk/texlive/linked_scripts/thumbpdf/thumbpdf.pl	2009-05-27 22:16:27.000000000 +0200
@@ -1,7 +1,4 @@
-eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}' && eval 'exec perl -S $0 $argv:q'
-  if 0;
-use strict;
-$^W=1; # turn warning on
+#!/usr/bin/perl -w
 #
 # thumbpdf.pl
 #
